client/js/lounge.js

```diff
@@ -994,6 +994,23 @@ $(function() {
 		});
 	});
 
+	$("#connect").one("show", function() {
+		var params = window.URI(document.location.search);
+		params = params.search(true);
+		// Possible parameters:  name, host, port, password, tls, nick, username, realname, join
+		for (var key in params) {
+			var value = params[key];
+			key = key.replace(/\W/g, "");
+
+			var element = $("#connect input[name='" + key + "']");
+			if (element.is(":checkbox")) {
+				element.prop("checked", (value === "1" || value === "true") ? true : false);
```

Comment:
This is setting the `checked` property. I could do an `if(is(':checked'))` but I don't think thats an improvement.